### PR TITLE
Fix - Update erigon repo

### PR DESCRIPTION
**Pull request by gummybera**
Erigon repo has moved from `thorax/erigon` to `erigontech/erigon`

## Summary by CodeRabbit

- **New Features**
	- Updated Docker image for the Erigon client across various configuration files to version `v2.60.9`, enhancing performance and compatibility.

- **Chores**
	- Cleaned up configuration files by removing outdated comments and references, improving clarity and organization.

### CODE REVIEW

**Review by abi87 — APPROVED** [review]
Thanks @gummybera. This solve some late issues with test-e2e on CI.
Cc @nidhi-singh02

File: kurtosis/src/nodes/nodes.star [hunk 1]
```diff
@@ -33,7 +33,7 @@ EXECUTION_DEFAULT_SETTINGS = {
     },
     "images": {
         "besu": "hyperledger/besu:latest",
-        "erigon": "thorax/erigon:v2.60.1",
+        "erigon": "erigontech/erigon:v2.60.1",
```
Comment (nidhi-singh02) [inline kurtosis/src/nodes/nodes.star:36]: We may want to change the docker version here, doc reference`Images before 2.60.7 are released in repository [“thorax/erigon”](https://hub.docker.com/r/thorax/erigon/tags) and will not be moved to the new repo;`. The new repo has images from 2.60.8 onwards.

File: kurtosis/beaconkit-all.yaml [hunk 2]
```diff
@@ -93,7 +93,7 @@ node_settings:
       max_memory: 2048
     images:
       besu: hyperledger/besu:24.5.4
-      erigon: thorax/erigon:v2.60.1
+      erigon: erigontech/erigon:v2.60.1
```
Comment (nidhi-singh02) [inline kurtosis/beaconkit-all.yaml:96]: The new repo has images from 2.60.8 onwards. Need to change the docker image version. Similar comments at other places as E2E tests are failing.

**Review by nidhi-singh02 — CHANGES_REQUESTED** [review]

File: kurtosis/beaconkit-all.yaml (hunk 2 above)
Comment (gummybera) [reply kurtosis/beaconkit-all.yaml]: thanks, bumped to the latest version wherever applicable.

**Review by nidhi-singh02 — APPROVED** [review]
lgtm!

**Pull request merged**